src/test/java/test/SimpleJavaScriptExecutorTest.java

```diff
@@ -0,0 +1,63 @@
+package test;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Platform;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import page.GoogleLoginPage;
+import page.GoogleMailHomePage;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class SimpleJavaScriptExecutorTest {
+
+    private WebDriver driver;
+    JavascriptExecutor javascriptExecutor;
+
+//    //    Browser setup for Selenium Grid
+//    @BeforeMethod(alwaysRun = true)
+//    public void browserSetup() {
+//        driver = new ChromeDriver();
+//        javascriptExecutor = (JavascriptExecutor) driver;
+//        driver.manage().deleteAllCookies();
+//        driver.manage().window().maximize();
+//    }
+
+//        Browser setup for Selenium Grid
+    @BeforeMethod(alwaysRun = true)
+    public void setDriver() {
+        System.setProperty("webdriver.chrome.driver", "D:\\BrowserDrivers\\chromedriver.exe");
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("start-maximized");
+        options.setCapability("platformName", Platform.WINDOWS);
+
+        try{
+            driver = new RemoteWebDriver(new URL("http://localhost:4444/wd/hub"), options);
+        } catch (MalformedURLException e) {
+            e.printStackTrace();
+        }
+        javascriptExecutor = (JavascriptExecutor) driver;
+        driver.manage().deleteAllCookies();
+    }
+
+    @Test (description = "Smoke")
+    public void testJavaScript() {
+        String login = "tyrmandyr1@gmail.com";
+        String password = "@Tyrmandyr1!";
+
+        GoogleMailHomePage homePage = new GoogleLoginPage(driver).loginToGoogleMail(login, password);
+        String currentUrl = javascriptExecutor.executeScript("return document.location.href;").toString();
+        Assert.assertEquals(currentUrl, homePage.getHomepageUrl(), "Login failed");
+        javascriptExecutor.executeScript("alert('Test Passed!');");
+        new WebDriverWait(driver, 5).until(ExpectedConditions.alertIsPresent());
+        driver.switchTo().alert().accept();
```

Comment:
The same as before

Reply:
Method generateTestPassAlert() created in GoogleMailHomePage class.